sfge/src/main/java/com/salesforce/graph/ops/expander/ApexPathExpanderUtil.java

```diff
@@ -67,18 +68,28 @@ private static void logFilteredOutPath(ThrowStatementVertex throwStatementVertex
         }
     }
 
-    /**
-     * The {@code ApexPathCollapser} that keeps track of all paths and attempts to collapse them
-     * whenever a forked method has returned a result.
-     */
-    private final ApexPathCollapser apexPathCollapser;
+    /** Used to give each object a unique id */
+    private static final AtomicLong ID_GENERATOR = new AtomicLong();
```

Comment:
Minor complaint: Can we move the property declarations up to the top?

Reply:
We probably need an inner class to hold these methods and fields. Let me see if I can keep the change simple enough.